src/core/compact_object.h

```diff
@@ -150,6 +151,27 @@ class CompactObj {
   using PrefixArray = std::vector<std::string_view>;
   using MemoryResource = detail::RobjWrapper::MemoryResource;
 
+  template <typename O>
```

Comment:
this is not the right place to add this function

Reply:
where do you use it?

Reply:
I was actually struggling with the placement of this, so thanks for the feedback.
This would be used in `hset_family` and `set_family`. I didn't see a clear way to facilitate the code-reuse that was suggested other than `compact_object`.

What would be a better fit?

Reply:
@dranikpg  just introduced family_utils.h - seems like a good place.